app/views/proposals/details/_status.html.haml

```diff
@@ -33,3 +33,6 @@
                         = step.status.capitalize + " "
                         %span.step-timestamp
                           = step.completed_at.strftime("%m/%d/%Y %I:%M%p")
+                    - elsif step.status == "actionable" || step.status == "pending"
```

Comment:
So, you can do `step.actionable?` instead of a string comparison here. But also you can possibly replace this test with `!step.completed?` since it's the only other status, isn't it?